src/lib/base-dao-mongoose.js

```diff
@@ -41,7 +41,13 @@ class BaseDaoMongoose {
   }
 
   async updateById(_id, data) {
-    return Promise.resolve(this.Model.findOneAndUpdate({ _id }, data));
+    const item = await Promise.resolve(
+      this.Model.findOneAndUpdate({ _id }, data)
```

Comment:
Should be lowercase. Uppercase variables are only constructor functions + classes

Reply:
Yes, it's mongoose class of model like User or Event.

Reply:
But you're not using it as a constructor anywhere

Reply:
Yes, I use it as constructor only in inherits classes, like UserDao.
Something like:
```
class UserDao extends BaseDao {
  async create(username, password, role) {
    const user = new this.Model();
    user.username = username;
    ...
    super.create(user)
    ...
}
```